src/test/java/io/netty/handler/codec/compression/BrotliDecoderTest.java

```diff
@@ -106,30 +136,12 @@ protected void testDecompression(final ByteBuf expected, final ByteBuf data) thr
   }
 
   @Test
-  public void testDecompressionOfBatchedFlowOfData() throws Exception {
```

Comment:
this test is replaced with `testDecompressionOfRandomlyChunkedData` which is more simple and correct